### PR TITLE
Using FAISS + 🤗 datasets with CLIP

### PR DESCRIPTION
**Pull request by merveenoyan**
This PR introduces the notebook for indexing and querying using FAISS + 🤗 datasets and CLIP model.

### CODE REVIEW

**Comment by merveenoyan** [comment]
@MKhalusova I addressed your comments <3

File: notebooks/FAISS_with_🤗_datasets_and_CLIP.ipynb [hunk 1]
```diff
@@ -0,0 +1,10919 @@
+{
```
Comment (MKhalusova) [inline notebooks/FAISS_with_🤗_datasets_and_CLIP.ipynb:1]: <p>"simply training a new model" =&gt; "simply train a new model"</p><p>"walks you through how to use" =&gt; "walks you through using"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (MKhalusova) [inline notebooks/FAISS_with_🤗_datasets_and_CLIP.ipynb:1]: <p>"we will <a href="https://huggingface.co/openai/clip-vit-base-patch16" target="_blank">CLIP model</a> " =&gt; "we will use <a href="https://huggingface.co/openai/clip-vit-base-patch16" target="_blank">CLIP model</a>"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (MKhalusova) [inline notebooks/FAISS_with_🤗_datasets_and_CLIP.ipynb:1]: <p>retro woman artwork? I'm not sure what this is. Aren't we looking for a beaver? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

**Review by MKhalusova — APPROVED** [review]
Thank you @merveenoyan ! LGTM

**Pull request merged**